src/lib/engine/collections.js

```diff
@@ -4,8 +4,12 @@ const equalsProp = (propValue, propName) => e => typeSafeEq(_.get(e, propName),
 const nullFn = x => null
 
 export const get = (collection, id, prop='id') => allPresent(collection, id) ? collection.find(equalsProp(id, prop)) : null
+export const gget = (collection, id, searchProp, getProp) => _.get(get(collection, id, searchProp), getProp)
```

Comment:
get from the result of a get. 

So, the way I read is, if you call:
```_collections.gget(metrics, 3, 'id', 'readableId')```
then that means "Get the property 'readableId' from the metric with 'id' '3' within the collection 'metrics'"